src/app/pages/i18n/i18n.component.scss

```diff
@@ -0,0 +1,112 @@
+@import '~@pxblue/colors/palette';
+.i18n-container {
+    height: 100vh;
+    width: 100%;
+}
+.sidenav {
+    max-width: 90%;
+    transition: width 200ms cubic-bezier(0.4, 0, 0.2, 1);
+    width: 350px;
+    .sidenav-content {
+        padding-top: 0;
+        overflow-x: hidden;
+    }
+    .toolbar-header {
+        background-color: map-get($pxb-blue, 500);
+        color: white;
+        min-height: 150px;
+    }
+    .logo-row {
+        height: 72px;
+        margin-left: 10px;
```

Comment:
can we clean up the margin/padding multiples of 5 and convert to multiples of 8?

Reply:
Yep, multiples of 8 now